src/celestia/timeinfo.h

```diff
@@ -0,0 +1,25 @@
+// timeinfo.h
+//
+// Copyright (C) 2023, the Celestia Development Team
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+
+#pragma once
+
+namespace celestia
+{
+
+struct TimeInfo
+{
+    static constexpr double MinimumTimeRate = 1.0e-15;
+
+    double currentTime{ 0.0 };
+    double fps{ 0.0 };
```

Comment:
do we really double precision for fps?

Reply:
Yep. https://github.com/CelestiaProject/Celestia/blob/c74cd97422538479a508aae9a580357e3c73b277/src/celestia/celestiacore.h#L531